browser/src/core/Debug.js

```diff
@@ -515,13 +517,28 @@ L.DebugManager = L.Class.extend({
 		}
 
 		// Toggle sidebar
+		if (!this._map._docLoadedOnce) {
+			// When first opening the document, initializeSidebar is called
+			// 200ms after setup, which would overwrite our randomization.
+			// So in this case, wait for sidebar initialization and the
+			// response to complete so that we know the current state
+			setTimeout(this._randomizeSidebar.bind(this), 1000);
+		} else {
+			this._randomizeSidebar();
+		}
+
+		this._painter.update();
```

Comment:
Does that do what you think it does? still no harm if it's only on a debug path.